### ISSUE TITLE
combine deferredScript with CDNResurceHandler generates CORS problem

### ISSUE THREAD

**Issue by jlerbsc** [omnifaces/omnifaces#356]
When we try to combine deferredScript with CDNResurceHandler we are facing to CORS problems.

Access to Script at 'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js' from origin 'https://xxxx' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

See Balusc response in StackOverflow http://stackoverflow.com/questions/42154883/omnifaces-deferred-script-with-cdnresourcehandler

**Comment by jlerbsc** [comment]
Balusc says that this technical problem is caused because this tag (<o:deferredScript>) explicitly sets crossorigin="anonymous" attribute in order to support script error reporting (so that JavaScript errors can be sent to the server side for logging). Your concrete problem will be solved when that attribute is not set.
You could report an issue to OmniFaces guys and tell them to skip that attribute when the script's src is external.

**Comment by BalusC** [comment]
This is after all not that trivial. I'm considering to just add a `cors="false"` attribute so you can control it from the tag on. Because, there may be CDNs which just have `Access-Control-Allow-Origin` correctly set.

**Comment by jlerbsc** [comment]
I'm using Omnifaces 1.14  could you do the modification in this version too?

**Comment by BalusC** [comment]
I decided to not implement it for 2 reasons:

1. The `o:deferredScript` was never intented to reference a non-JSF resource. The `CDNResourceHandler` also not, by the way. They are both primarily designed to defer or CDN an automatically included JSF resource.
2. There's no advantage over using "plain HTML" in this specific case. In this specific case a `<script async="true" src="...">` works just fine.
